server/tests/api_browser_test.py

```diff
@@ -126,11 +126,36 @@ def test_get_statvar_hierarchy(self, mock_sv_groups):
             "group4": ["group1", "group3"],
             "group5": []
         }
+        expected_sv_levels = {
```

Comment:
can you make one object for the entire expected result, that way no need to assert for each field

Reply:
done